package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -405,22 +405,22 @@ The most straightforward way to concatentate two AtomGroups is by using the
 ``+`` operator::
 
  >>> ordered = u.select_atoms("segid DMPC and resid 3 and name P") + u.select_atoms("segid DMPC and resid 2 and name P")
- >>> print list(ordered)
+ >>> print list(ordered) 
```

Comment:
Please, remove this additional space:
```suggestion
 >>> print list(ordered)
```

Reply:
@HenokB note that you could have use the "Commit suggestion" to just make the changes in the PR that @RMeli so helpfully included. It's totally fine to do it manually but I wanted to point out some of the available tools to be efficient.

Reply:
> @HenokB note that you could have use the "Commit suggestion" to just make the changes in the PR that @RMeli so helpfully included. It's totally fine to do it manually but I wanted to point out some of the available tools to be efficient.

@orbeckst thanks, I actually found out about "commit suggestions" after doing it manually, and I've to say that they are really efficient and simple.